app/src/androidTest/java/ch/epfl/sweng/favors/utils/UtilsTest.java

```diff
@@ -132,9 +136,28 @@ public void passwordWithoutRequirements(){
     public void failingToast(){
         ActivityTestRule<TestActivity> activityActivityTestRule = new ActivityTestRule<>(TestActivity.class);
         activityActivityTestRule.launchActivity(null);
-        Looper.prepare();
+
+        if (Looper.myLooper() == null)
+        {
+            Looper.prepare();
+        }
 
         Utils.displayToastOnTaskCompletion(Tasks.forCanceled(),activityActivityTestRule.getActivity(),"success","failure");
     }
 
+    @Test
+    public void canLogout(){
+        ActivityTestRule<TestActivity> activityActivityTestRule = new ActivityTestRule<>(TestActivity.class);
+        activityActivityTestRule.launchActivity(null);
+
+        if (Looper.myLooper() == null)
+        {
+            Looper.prepare();
+        }
+
+        Authentication auth = mock(Authentication.class);
+
+        Utils.logout(activityActivityTestRule.getActivity(), auth);
```

Comment:
can we check to make sure that we are properly logout?

Reply:
Thought about it, gonna try to do it if there's any time left otherwise adding it for next sprint